core/src/main/scala/org/http4s/internal/parsing/Rfc2616.scala

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2013-2020 http4s.org
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.http4s.internal.parsing
+
+import cats.parse.Parser1
+import cats.parse.Parser.{char, charIn, string1}
+import cats.parse.Rfc5234.{digit, sp}
+
+/** Common rules defined in RFC2616.  This RFC is now obsolete,
+  * but some other active ones still refer to it.
+  *
+  * @see [[https://tools.ietf.org/html/rfc2616#section-2.2] RFC2616, Basic Rules]
+  */
+private[http4s] object Rfc2616 {
+  /* CHAR           = <any US-ASCII character (octets 0 - 127)>
+   * CTL            = <any US-ASCII control character
+   *                  (octets 0 - 31) and DEL (127)>
+   * separators     = "(" | ")" | "<" | ">" | "@"
+   *                | "," | ";" | ":" | "\" | <">
+   *                | "/" | "[" | "]" | "?" | "="
+   *                | "{" | "}" | SP  | HT
+   * token          = 1*<any CHAR except CTLs or separators>
+   */
+  val token: Parser1[String] = {
```

Comment:
I haven't done the math to figure out if this is equivalent to our other token, but this way, it's expressed fairly literally compared to the referenced RFC.